client/app/utils/jsonSchema/operations.ts

```diff
@@ -1,59 +1,373 @@
+    "physical_postal_code",
+    "legal_land_description",
     "latitude",
     "longitude",
-    "legal_land_description",
-    "nearest_municipality",
-    "operator_percent_of_ownership",
-    "estimated_emissions",
+    "npri_id",
+    "bcer_permit_id",
   ],
   properties: {
-    operator_id: { type: "number", title: "operator_id" },
-    name: { type: "string", title: "name" },
-    operation_type: { type: "string", title: "operation_type" },
-    naics_code_id: { type: "number", title: "naics_code", enum: [] },
-    eligible_commercial_product_name: {
+    verified_by: { type: "string" },
+    verified_at: { type: "string" },
+    name: { type: "string", title: "Operation Name" },
+    type: { type: "string", title: "Operation Type" },
+    naics_code_id: { type: "number", title: "NAICS Code" },
+    naics_category_id: { type: "number", title: "NAICS Category" },
+    reporting_activities: { type: "string", title: "Reporting Activities" },
+    permit_issuing_agency: { type: "string", title: "Permit Issuing Agency " },
+    permit_number: { type: "string", title: "Permit Number" },
+    "Did you submit a GHG emissions report for reporting year 2022?": {
+      type: "boolean",
+      default: false,
+    },
+    physical_street_address: { type: "string", title: "Physical Address" },
+    physical_municipality: { type: "string", title: "Municipality" },
+    physical_province: {
+      type: "string",
+      title: "Province",
+      enum: [
+        "NL",
+        "PE",
+        "NS",
+        "NB",
+        "QC",
+        "ON",
+        "MB",
+        "SK",
+        "AB",
+        "BC",
+        "YT",
+        "NT",
+        "NU",
+      ],
+    },
+    physical_postal_code: {
       type: "string",
-      title: "eligible_commercial_product_name",
-    },
-    permit_id: { type: "string", title: "permit_id" },
-    npr_id: { type: "string", title: "npr_id" },
-    ghfrp_id: { type: "string", title: "ghfrp_id" },
-    bcghrp_id: { type: "string", title: "bcghrp_id" },
-    petrinex_id: { type: "string", title: "petrinex_id" },
-    latitude: { type: "number", title: "latitude" },
-    longitude: { type: "number", title: "longitude" },
-    legal_land_description: { type: "string", title: "legal_land_description" },
-    nearest_municipality: { type: "string", title: "nearest_municipality" },
-    operator_percent_of_ownership: {
-      type: "number",
-      title: "operator_percent_of_ownership",
-    },
-    estimated_emissions: { type: "number", title: "estimated_emissions" },
+      title: "Postal Code",
+      maxLength: 7,
+    },
+    legal_land_description: { type: "string", title: "Legal Land Description" },
+    latitude: { type: "number", title: "Lat coordinates" },
+    longitude: { type: "number", title: "Long coordinates" },
+    npri_id: { type: "number", title: "NPRI ID" },
+    bcer_permit_id: { type: "number", title: "BCER permit ID" },
+    "Does the operation have multiple operators?": {
+      type: "boolean",
```

Comment:
The RJSF default theme does have a radio button option for boolean fields, but it's not working correctly. For now I just used checkboxes since we'll need to do a custom theme later anyway